nav2_behavior_tree/nav2_tree_nodes.xml

```diff
@@ -41,6 +41,14 @@
       <output_port name="output_goals">Set of goals after removing any passed</output_port>
     </Action>
 
+    <Action ID="RemovePassedGoalsWithMultiplePlanners">
+      <input_port name="input_goals">Input goals to remove if passed</input_port>
+      <input_port name="input_planner_ids">Input planner ids for set of goals</input_port>
+      <input_port name="radius">Radius tolerance on a goal to consider it passed</input_port>
```

Comment:
I cannot find the global_frame and the robot_frame port

Reply:
Looks to me like an oversight